dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -781,6 +781,28 @@ private static CompletionsOptions CreateCompletionsOptions(string text, OpenAIPr
                         break;
                 }
                 break;
+
+            case JsonElement formatElement:
+                // This is a workaround for a type mismatch when deserializing a JSON into an object? type property.
+                // Handling only string formatElement.
+                if (formatElement.ValueKind == JsonValueKind.String)
+                {
+                    string? formatString = formatElement.GetString();
```

Comment:
```suggestion
                    string formatString = formatElement.GetString() ?? "";
```
I think this is slightly better than using if statements for null checks.